src/API/Options.php

```diff
@@ -64,27 +64,80 @@ public function register_routes() {
 	}
 
 	/**
-	 * Check if a given request has access to manage options.
+	 * Check if a given request has access to get options.
+	 *
+	 * @param  WP_REST_Request $request Full details about the request.
+	 * @return WP_Error|boolean
+	 */
+	public function get_item_permissions_check( $request ) {
+		$params      = explode( ',', $request['options'] );
+		$permissions = $this->get_option_permissions( $request );
+
+		if ( ! is_array( $params ) ) {
+			return new \WP_Error( 'woocommerce_rest_cannot_view', __( 'You must supply an array of options.', 'woocommerce-admin' ), 500 );
+		}
+
+		foreach ( $params as $option ) {
+			if ( ! isset( $permissions[ $option ] ) || ! $permissions[ $option ] ) {
+				return new \WP_Error( 'woocommerce_rest_cannot_update', __( 'Sorry, you cannot manage these options.', 'woocommerce-admin' ), array( 'status' => rest_authorization_required_code() ) );
```

Comment:
Minor: Error code mentions `update` but this is a `GET` endpoint.

Reply:
Nice catch, updated in https://github.com/woocommerce/woocommerce-admin/pull/3004/commits/ba72b7eb457ba83a6b14966c768ba3d2ea8b8c99